src/generators/Sequence.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Created by chetanv on 01/06/16.
+ */
+
+import Generator from './Generator';
+
+class Sequence extends Generator {
+  static sequences = {};
+  id = '';
+
+  constructor(factoryGirl, id, callback = null) {
```

Comment:
Yes, there was a reason! We need ids because of the initialiser function. In case of initialiser function, the function will get executed for every attrs/build/create call, resulting in a new sequence generator instance every time (that would start with 0 :( ). The id is used to keep track of the current sequence value.

In case we don't use ids, users will need to create the sequence instance somewhere outside the initialiser function, and that seemed error prone to me. 

I too liked sequences without ids, and it would keep simple for users to migrate as well. If you can suggest a workaround, I'll be glad to implement it.